### DISCUSSION TITLE
HTTPS in HFS

### DISCUSSION THREAD

**Discussion by kenrmayfield** [rejetto/hfs#120] [General]
Do you still have to Setup STunnel in order to have HTTPS running with HFS?   Can you just change the Port to 443 in HFS and Delete or Disable Port 80 in HFS for HTTPS to be used?

Thanks,
Kenneth

**Comment by rejetto** [comment]
HFS 3 supports HTTPS and needs no stunnel.
You can have only https, only http, or both.

**Comment by kenrmayfield** [comment]
Massimo,

Thank you for the Quick Response.   Sorry I am a Newbi to HFS.

Where can I Download HFS3 for Windows?

On another Post I Posted about the Layout Theme from 2.3m.......I was 
just asking if that Layout Theme
can be used with other Versions of HFS?

So 2.4.0RC6 is HFS2?

Excellent Work on HFS you have done.   I like this better than Dropbox 
or others.

Thanks,
Kenneth

On 2023-02-27 03:38 AM, Massimo Melina wrote:

> HFS 3 supports HTTPS and needs no stunnel.
> You can have only https, only http, or both.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/120#discussioncomment-5120750
[2]

**Comment by rejetto** [comment]
find hfs3 for windows at https://github.com/rejetto/hfs/releases
when I'll feel ready I'll replace it on the "official website" as well.

> So 2.4.0RC6 is HFS2?

yes, as an abbreviation for any version starting with "2"

> Excellent Work on HFS you have done.   I like this better than Dropbox or others.

thanks! They are very different anyway, I don't consider hfs to be a competitor of dropbox.

**Comment by kenrmayfield** [comment]
Massimo,

I did not mean any Disrespect when I mention Dropbox.

I just like your HFS better.

Thanks,
Kenneth

On 2023-02-27 08:04 AM, Massimo Melina wrote:

> find hfs3 for windows at https://github.com/rejetto/hfs/releases
> when I'll feel ready I'll replace it on the "official website" as well.
> 
>> So 2.4.0RC6 is HFS2?
> 
> yes, as an abbreviation for any version starting with "2"
> 
>> Excellent Work on HFS you have done. I like this better than Dropbox 
>> or others.
> 
> thanks! They are very different anyway, I don't consider hfs to be a 
> competitor of dropbox.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/120#discussioncomment-5123173
[2]

**Comment by rejetto** [comment]
but of course, i know, I just commented because I find it relevant 🙂

**Comment by kenrmayfield** [comment]
Massio,

I just tried 0.37.0 and tried to Turn Off HTTP and Turn On HTTPS.  I get 
a Error Message that you can not
Turn Off Both.

I am not trying to Turn Off Both but to Turn On HTTPS Only.

Thanks,
Kenneth

On 2023-02-27 03:38 AM, Massimo Melina wrote:

> HFS 3 supports HTTPS and needs no stunnel.
> You can have only https, only http, or both.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/120#discussioncomment-5120750
[2]

**Comment by rejetto** [comment]
at the moment the only way is to first turn on https (and save) then you switch off http

**Comment by rejetto** [comment]
since you are initially accessing the admin-panel over http,
after you turn on https, please access the admin panel on https.
otherwise when you switch off http it will stop working.
In the future i will make this automatic.

**Reply by rejetto** [reply]
i'm glad to say that this inconvenience will be solved by next release

**Comment by kenrmayfield** [comment]
Massimo,

It is not a Inconvenience.   You are doing Excellent Work!!!

I  Appreciate Your Time and Responses.

On 2023-03-01 04:50 AM, Massimo Melina wrote:

> i'm glad to say that this inconvenience will be solved by next release
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/rejetto/hfs/discussions/120#discussioncomment-5165413
[2]
